x/dao/keeper/msg_server_distribution_result.go

```diff
@@ -53,7 +59,7 @@ func (k msgServer) resolveStagedClaims(ctx sdk.Context, start int64, end int64)
 		keys = append(keys, p)
 	}
 	for _, p := range keys {
-		err = k.convertClaim(ctx, p, popParticipants[p])
+		err = k.convertAccountClaim(ctx, p, popParticipants[p])
 		if err != nil {
 			return err
 		}
```

Comment:
Shouldn't we log which accounts couldn't be converted and continue to convert them until we reach the end of the complete distribution approach?

Reply:
I would rather return the error and let the entire message fail as this would result in the deliverState not being commited to root state / disk. We could then rerun the distribution result with the provided pop-height from the log.

See here: https://docs.cosmos.network/v0.47/learn/advanced/baseapp#state-updates (especially DeliverTx State Updates & Commit State Updates)